docs/root/configuration/http/http_filters/bandwidth_limit_filter.rst

```diff
@@ -0,0 +1,64 @@
+.. _config_http_filters_bandwidth_limit:
+
+Bandwidth limit
+====================
+
+* Bandwidth limiting :ref:`architecture overview <arch_overview_bandwidth_limit>`
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`
+* This filter should be configured with the name ``envoy.filters.http.bandwidth_limit``.
+
+The HTTP Bandwidth limit filter limits the size of data flow to the max bandwidth set in the ``limit_kbps``
+when the request's route or virtual host has a per filter
+:ref:`bandwidth limit configuration <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`.
+
+If the bandwidth limit has been exhausted the filter stops further transfer until more bandwidth gets allocated
+according to the ``fill_interval`` (default is 50 milliseconds). If the connection buffer fills up with accumulated
+data then the source of data will be readDisabled as described in the :repo:`flow control doc<source/docs/flow_control.md>`.
+
+.. note::
+  The token bucket is shared across all workers, thus the limits are applied per Envoy process.
+
+Example configuration
+---------------------
+
+Example filter configuration for a globally disabled bandwidth limiter but enabled for a specific route:
+
+.. literalinclude:: _include/bandwidth-limit-filter.yaml
+    :language: yaml
+	:lines: 11-53
```

Comment:
i think this is the mixed tabs/spaces fail - https://dev.azure.com/cncf/envoy/_build/results?buildId=76487&view=logs&j=c5dd2866-6ab3-5f3c-3a44-4cef0ec909b5&t=a9eb66d6-8944-5769-b3f7-476949dadcb8&l=60

you probably want your editor to autofix-on-save the glint checks

Reply:
thanks